league/templates/league/players.html

```diff
@@ -67,7 +67,7 @@
           {% for division in divisions %}
             <div class="checkbox div-checkbox ">
               <label>
-                <input class ="check-div"  onchange="filterdivs()" value="{{division}}" type="checkbox" checked>{{division.name}}
+                <input class ="check-div"  onchange="filterdivs()" value="{{division.name}}" type="checkbox" checked>{{division.name}}
```

Comment:
This is an unrelated fix right?

Reply:
yep, the Datatable filtering didn't work because of that

Reply:
I though it was not worth to mention

Reply:
Nice catch!

Reply:
And i just see i forgot the top one ^^